lib/composables/Http.ts

```diff
@@ -0,0 +1,141 @@
+import { parse as parseContentDisposition } from '@tinyhttp/content-disposition'
+import { parse as parseCookie } from '@tinyhttp/cookie'
+import FileSaver from 'file-saver'
+import { $fetch } from 'ofetch'
+import { stringify } from 'qs'
+
+type FetchOptions = Parameters<typeof $fetch>[1]
+
+export class Http {
+  xsrfUrl: string
```

Comment:
How about making this static? So that we only have to set it once in `main.ts` for example 👀 (plus default value maybe).

```ts
export class Http {
  static xsrfUrl: string = '/api/sanctum/csrf-cookie'
  ...
}

// In `main.ts`
import { Http } from 'sefirot/http/Http'

Http.xsrfUrl = '/api/custom-path'

// When using
new Http().get(...) // no need for constructor args
```